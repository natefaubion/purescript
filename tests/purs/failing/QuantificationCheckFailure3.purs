-- @shouldFailWith VisibleQuantificationCheckFailureInType
module Main where

foreign import data KProxy :: forall (k :: Type) . k -> Type
foreign import data TProxy :: forall (k :: Type) (t :: k) . KProxy t

type Hmm k = (TProxy :: KProxy k)
