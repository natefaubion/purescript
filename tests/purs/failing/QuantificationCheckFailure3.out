Error found:
in module [33mMain[0m
at tests/purs/failing/QuantificationCheckFailure3.purs:7:1 - 7:34 (line 7, column 1 - line 7, column 34)

  Visible dependent quantification of type variable [33mk[0m is not supported.
  If you would like this feature supported, please bother Liam Goodacre (@LiamGoodacre).

in type synonym [33mHmm[0m

See https://github.com/purescript/documentation/blob/master/errors/VisibleQuantificationCheckFailureInType.md for more information,
or to contribute content related to this error.

